src/main/java/se/ayoy/maven/plugins/licenseverifier/LicenseVerifierMojo.java

```diff
@@ -228,8 +228,9 @@ private void determineArtifactStatus(
                 logInfoIfVerbose("    Got licenseInfo with status : " + info.getStatus());
                 artifactToCheck.addLicenseInfo(info);
 
-                determineArtifactStatus(childNode, licenseInfoFile);
             }
+            determineArtifactStatus(childNode, licenseInfoFile);
```

Comment:
Add an empty line before the determineArtifactStatus method call and remove the one after.